src/components/Sidebar/BrowseContainer.js

```diff
@@ -0,0 +1,9 @@
+import { connect } from 'react-redux';
+import Browse from './Browse';
+
+function mapStateToProps(state) {
```

Comment:
You can ignore coverage for this, as indicated in https://github.com/Joonpark13/serif.nu/blob/master/docs/CONTRIBUTING.md#unit-testing